scss/bitstyles/organisms/page-header/_index.scss

```diff
@@ -0,0 +1,66 @@
+@use '../../tools/media-query';
+@use '../../tools/classname';
+@use '../../tools/design-token';
+
+#{classname.get($classname-items: 'page-header', $layer: 'organism')} {
+  padding-top: var(design-token.get('size', 'm'));
+  background-color: var(design-token.get('color', 'grayscale', 'light-3'));
+}
+
+.o-page-header__content {
+  @include media-query.get('m') {
+    display: grid;
+    grid-template-columns: repeat(2, minmax(0, 1fr));
+  }
+}
+
+.o-page-header__top-left {
+  @include media-query.get('m') {
+    grid-column: 1;
```

Comment:
Using `grid-template-area` may make for more readable CSS here, wdyt? (https://css-tricks.com/snippets/css/complete-guide-grid/#aa-grid-template-areas)